docs/getting_started.md

```diff
@@ -0,0 +1,135 @@
+
+These libraries are published to Rubygems.org, so you can add Decidim to your Ruby on Rails app as external dependencies.
+
+If you want to start your own installation of Decidim, you don't need to clone this repo. Keep reading to find out how to install Decidim.
+
+## Creating your Decidim app
+
+### Using Docker [experimental]
+
+> *Please note that this is **experimental***
+
+Make sure you [have Docker v17 at least](https://docs.docker.com/engine/installation/). `cd` to your preferred folder and run this command:
+
+```
+docker run --rm -v $(pwd):/tmp codegram/decidim bash -c "bundle exec decidim /tmp/decidim_application"
+```
+
+This will create a `decidim_application` Ruby on Rails app using Decidim in the current folder. It will install the latest released version of the gem.
+
+### Step by step
+
+First of all, you need to install the `decidim` gem:
+
+```
+$ gem install decidim
+```
+
+Afterwards, you can create an application with the nice `decidim` executable:
+
+```
+$ decidim decidim_application
+$ cd decidim_application
+```
+
+### Initializing your app for local development
+
+You should now setup your database:
+
+```
+$ bin/rails db:create db:migrate db:seed
+```
+
+This will also create some default data so you can start testing the app:
+
+* A `Decidim::System::Admin` with email `system@example.org` and password `decidim123456`, to log in at `/system`.
+* A `Decidim::Organization` named `Decidim Staging`. You probably want to change its name and hostname to match your needs.
+* A `Decidim::User` acting as an admin for the organization, with email `admin@example.org` and password `decidim123456`.
+* A `Decidim::User` that also belongs to the organization but it's a regular user, with email `user@example.org` and password `decidim123456`.
+
+This data won't be created in production environments, if you still want to do it, run:
+
+```
+$ SEED=true rails db:setup
+```
+
+You can now start your server!
+
+```
+$ bin/rails s
+```
+
+Visit [http://localhost:3000](http://localhost:3000) to see your app running.
+
+## Configuration & setup
+
+Decidim comes pre-configured with some safe defaults, but can be changed through the `config/initializers/decidim.rb` file in your app. Check the comments there or read the comments in [the source file](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/decidim-core/lib/decidim/core.rb) (the part with the `config_accessor` calls) for more up-to-date info.
+
+We also have other guides on how to configure some extra features: 
+
+- [Social providers integration](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/docs/social_providers.md): Enable sign up from social networks.
+- [Analytics](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/docs/analytics.md): How to enable analytics
+- [Geocoding](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/docs/geocoding.md): How to enable geocoding for proposals and meetings
+
```

Comment:
I'd work on this once we make a PR to add docs on how to develop on top of Decidim, since this PR is only about getting your app up and running.